components/event-chart/src/_event-chart-constants.scss

```
@@ -1,2 +1,20 @@
 $dt-event-chart-event-height: 16px;
 $dt-event-chart-lane-height: $dt-event-chart-event-height * 3;
+$dt-event-chart-hover-spread: 16px;
```

The stroke width is applied to an svg element. To keep  scales intact, it would be beneficial to not apply pixel values but rather viewbox relative units.

```suggestion
$dt-event-chart-hover-spread: 16;
```

---

AS we are applying a 1:1 mapping between the pixel width of the chart container and the svg viewbox, this one does not make a visual difference, but if you get around to changing it, this would be nice.